src/util/rocksdb/mod.rs

```diff
@@ -134,11 +162,22 @@ fn check_and_open(
     let existed: Vec<&str> = cfs_list.iter().map(|v| v.as_str()).collect();
     let needed: Vec<&str> = cfs_opts.iter().map(|x| x.cf).collect();
 
+    let cf_descs = if !existed.is_empty() {
+        // panic if OPTIONS not found for existing instance?
+        let (_, tmp) = load_latest_options(path, &Env::default(), false)
```

Comment:
why two unwrap here...

Reply:
because the return value is something like Result<Options<>, String>